### PR TITLE
Fix errors reported by PHPStan

### PR DESCRIPTION
**Pull request by szepeviktor**
## What?

From #618

## Testing Instructions

```
vendor/bin/phpstan analyze -c vendor/szepeviktor/phpstan-wordpress/extension.neon includes/ providers/ class-*.php two-factor.php --level=0
```

### CODE REVIEW

File: providers/class-two-factor-fido-u2f-admin.php [hunk 1]
```diff
@@ -230,7 +230,7 @@ public static function show_user_profile( $user ) {
 	 * @static
 	 *
 	 * @param int $user_id User ID.
-	 * @return false
+	 * @return void|never
```
Comment (szepeviktor) [inline providers/class-two-factor-fido-u2f-admin.php:233]: This is dubious.
`catch` should do better.

Comment (dd32) [reply providers/class-two-factor-fido-u2f-admin.php:233]: This file is pending removal via #439 so not too much effort needs to happen here :)

**Review by kasparsd — APPROVED** [review]
This is a nice cleanup! Looks good!

**Comment by szepeviktor** [comment]
Glad to contribute.

**Pull request merged**